src/core/hle/libraries/libc/libc_math.cpp

```diff
@@ -4,35 +4,35 @@
 namespace Core::Libraries::LibC {
 
 float PS4_SYSV_ABI ps4_atan2f(float y, float x) {
-    return std::atan2f(y, x);
```

Comment:
Any particular reason for dropping the std namespace on these

Reply:
I'm not sure why but the compiler generated errors that these functions do not belong in namespace std, so I removed it for now. From searching online some say it's a libstdc++ bug but that's a bit different to believe.